src/main/java/com/gamingmesh/jobs/CMILib/CMIMaterial.java

```diff
@@ -1410,10 +1410,7 @@ public static CMIMaterial get(String id) {
 		}
 		CMIMaterial mat1 = ItemManager.byName.get(id);
 		if (mat1 != null && mat1.getLegacyId() > 0) {

```

Comment:
What about the second legacyId condition?

Reply:
Was leftover code from before I removed the previous return. The intention was to just fix legacy materials being returned. Removing the second condition would cause the wrong material to be returned in more circumstances, no?

Reply:
Not quite. LegacyId is not used, so it does nothing but check to see if it is greater than 0. This should be used for this in a second check as well before the `mat1` condition to make sure the legacyId is properly checked.

An example how I mean:
```java
if (mat.getLegacyId() > 0 && (mat = get(mat1.getLegacyId(), data)) != null) {
    return mat;
}

CMIMaterial mat1 = ItemManager.byName.get(id);
if (mat1 != null) {
...
```

Reply:
Just added another commit to fix this, made sure to invert the conditions too.